frontend/src/app/main/ui/ds/controls/utilities/input_field.cljs

```diff
@@ -32,22 +32,23 @@
    ::mf/forward-ref true
    ::mf/schema schema:input-field}
   [{:keys [id icon has-hint hint-type class type max-length variant children] :rest props} ref]
-  (let [ref   (or ref (mf/use-ref))
+  (let [input-ref (mf/use-ref)
         type  (d/nilv type "text")
         variant (d/nilv variant "dense")
         props (mf/spread-props props
                                {:class (stl/css-case
                                         :input true
-                                        :input-with-icon (some? icon))
-                                :ref ref
+                                        :input-with-icon (some? icon)
+                                        )
+                                :ref (or input-ref ref)
                                 :aria-invalid (when (and has-hint
                                                          (= hint-type "error"))
                                                 "true")
                                 :aria-describedby (when has-hint
                                                     (str id "-hint"))
                                 :type (d/nilv type "text")
                                 :id id
-                                :maxLength (d/nilv max-length (str max-input-length))})
+                                :max-length (d/nilv max-length (str max-input-length))})
```

Comment:
you can remove the `(str` part from max-input-length i guess

Reply:
Since the input element expects a string I tried to use the expected type but works as well without converting it manually@